applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -226,7 +229,19 @@ - (void)initializeWithPluginVersion:(NSString *)pluginVersion sdkKey:(NSString *
         self.sdk.targetingData.interests = self.targetingInterestsToSet;
         self.targetingInterestsToSet = nil;
     }
-    
+
+    if ( self.extraParametersToSet.count > 0 )
+    {
+        ALSdkSettings *settings = [ALSdk shared].settings;
+        for (NSString* key in self.extraParametersToSet)
+        {
+            NSString* value = self.extraParametersToSet[key];
+            [settings setExtraParameterForKey: @"uid2_token" value: @"VALUE"];
+        }
+        [self.extraParametersToSet removeAllObjects];
+    }
+
```

Comment:
Same as Android.   Can yo implement [setPendingExtraParametersIfNeeded](https://github.com/AppLovin/AppLovin-MAX-React-Native/blob/e471e4db559cecd72daabe4f70903c7c77deafa9/ios/AppLovinMAX.m#L1880C9-L1880C42)?  Also, remove your key and value.